src/emulator/auth/state.ts

```diff
@@ -74,7 +84,7 @@ export class ProjectState {
     const user = this.updateUserByLocalId(localId, props, {
       upsertProviders: props.providerUserInfo,
     });
-    this.authCloudFunction.dispatch("create", user);
+    this._authCloudFunction?.dispatch("create", user);
```

Comment:
Should triggers work for tenant projects? Not sure about this, but my guess is yes. (When in doubt try it out in production)